client/src/komponenter/Infographic/InngangTilSykefraværsstatistikk.tsx

```diff
@@ -1,31 +1,18 @@
 import { Lenkeflis } from "../../Lenkeflis/Lenkeflis";
-import { StatistikkIkon } from "../../Forside/ikoner/StatistikkIkon";
-import { BodyLong } from "@navikt/ds-react";
-import styles from "./Infographic.module.scss";
-import { LenkeMedEventutsendelse } from "../../LenkeMedNavigereEvent/LenkeMedEventutsendelse";
 
 export const InngangTilSykefraværsstatistikk = (props: {
   sykefravarsstatistikkUrl: string;
   useMobileVersion: boolean;
 }) => {
-  if (props.useMobileVersion) {
-    return (
-      <Lenkeflis
-        overskrift={"Sykefraværs&shy;statistikk"}
-        ikon={<StatistikkIkon />}
-        brødtekst={""}
-        href={props.sykefravarsstatistikkUrl}
-        fyltoppBakgrunn={true}
-      />
-    );
-  }
   return (
-    <BodyLong className={styles.oversiktTekst} size="medium">
-      Trenger du en større oversikt?{" "}
-      <LenkeMedEventutsendelse
-        href={props.sykefravarsstatistikkUrl}
-        lenketekst="Klikk her for å gå til statistikksiden."
-      />
-    </BodyLong>
+    <Lenkeflis
+      overskrift={"Se statistikk"}
+      brødtekst={
+        "Ved å sammenligne dere med andre og vite årsakene til fraværet, kan dere forebygge og redusere sykefravær."
```

Comment:
hvor kommer denne teksten fra? sier statistikksiden noe om årsakene til fraværet, utover den hardkodede teksten om at muskel- og skjelettplager er den vanligste årsaken i Norge?

Reply:
Den hadde vi hard code fra før, den endrer seg nesten sjelden i praksis, mens i teorien ville det endret seg hvert kvartal. Så jeg synes at det ikke er verdt å bruke energi for å gjøre det til mer kompleks per i dag. 
Kanskje senere blir det hentet ut fra DVH.